interpreter/runtime/event.ml

```diff
@@ -0,0 +1,10 @@
+open Types
```

Comment:
Even if its kind of trivial, can you also add an .mli file for this module?

Reply:
Done